logger.go

```diff
@@ -75,9 +78,15 @@ func Init(name string, verbose, systemLog bool, logFile io.Writer) *Logger {
 	}
 
 	var l Logger
-	l.infoLog = log.New(io.MultiWriter(iLogs...), "INFO: ", flags)
-	l.errorLog = log.New(io.MultiWriter(eLogs...), "ERROR: ", flags)
-	l.fatalLog = log.New(io.MultiWriter(eLogs...), "FATAL: ", flags)
+	l.infoLog = log.New(io.MultiWriter(iLogs...), tagInfo, flags)
+	l.infoLog = log.New(io.MultiWriter(iLogs...), tagWarning, flags)
```

Comment:
l.warningLog

Reply:
Also info only logs to stdout if verbose is set, does this make sense for warning too?

Reply:
Good catch.  Done.

By default glog logs `ERROR` and above to stderr, and `-alsologtostderr` to drop the threshold to `INFO`, so they are consistent.  However as this package doesn't really have a verbosity control, so there is no good way to align them.